com.github.you_win.VPupPr.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppdataEdit 4.0-->
+  <id>com.github.you_win.VPupPr</id>
```

Comment:
So I will repeat what I said. This does not match the project URL.

You download from `https://github.com/virtual-puppet-project/vpuppr`
and this is also what is in the URL fields in the appstream file.

Reply:
I said it **didn't match the project**. There was NOT A SINGLE REFERENCE to 'youwin' beside the app-id.

Reply:
I thought the project was still available at the you-win/vpuppr link, so I put that. I've fixed it, and will push the commits soon.